packages/neural_analytics_model/src/utils/trainer.py

```
@@ -42,39 +87,53 @@ def accuracy_torch(outputs, targets):
     
     return correct / total
 
-def train_model(train_loader, device, writer, epochs=1000, learning_rate=0.001):
+def train_model(train_loader, val_loader, device, writer, epochs=100, learning_rate=0.001, label_smoothing=0.0, class_weights=None):
```

Ruff (0.14.6)

Unused function argument: `label_smoothing`

(ARG001)